extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLog.i18n.php

```diff
@@ -26,6 +26,7 @@
 	'discussionslog-app-header' => 'App',
 	'discussionslog-site-header' => 'Site',
 	'discussionslog-timestamp-header' => 'Timestamp',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Timestamp" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "discussionslog-timestamp-header" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)